src/strands/multiagent/a2a/server.py

```diff
@@ -32,6 +32,7 @@ def __init__(
         host: str = "0.0.0.0",
         port: int = 9000,
         version: str = "0.0.1",
+        skills: list[AgentSkill] | None,
```

Comment:
Let's default this to `None` so that you don't have to pass it unless you want to.  It should also clean up the tests.

```suggestion
        skills: Optional[list[AgentSkill]] = None
```